binder/views.py

```diff
@@ -74,6 +75,11 @@ def image_transpose_exif(im):
 		return im
 
 
+def getsubclasses(cls):
```

Comment:
Note that in case of a diamond structure, classes will be returned more than once here. Was this intentional? In that case I would add a comment on why this was done.

Reply:
Simplicity basically, I could check for this but I think this would make the function more complicated and I don't expect classes to have that many subclasses that this would become a performance issue. 2 level's of inheritance within models would probably already be very rare.